borg/helpers.py

```diff
@@ -487,10 +501,6 @@ def format_archive(archive):
     return '%-36s %s' % (archive.name, to_localtime(archive.ts).strftime('%c'))
 
 
-class IntegrityError(Error):
```

Comment:
so both classes deriving from Error are together. before it was just at some random position.